app/components/dashboard/page_content_component.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+module Dashboard
+  # Renders an optional user message on the dashboard page
+  class PageContentComponent < ApplicationComponent
```

Comment:
Can you explain why there are multiple variations of the PageContentComponent?

Reply:
One is for the home page, the other is for the dashboard.  The styling is different between them.

We could use a single component, but there would be some additional logic required, so tradeoff between simpler component with some duplication (current), vs less duplication but more complicated (potential change).